finetuning/evaluation/util.py

```diff
@@ -80,12 +80,10 @@ def get_dataset_paths(dataset_name, split_choice):
     return raw_dir, labels_dir
 
 
-def get_model(model_type, ckpt, lora_rank):
```

Comment:
@anwai98 This function should be unnecessary and can be replaced by directly calling get_sam_model

Reply:
Yep, makes sense to get rid of it. I took care of it. Will push the changes in the new PR soon.